db/warehouse/migrate/20230608152942_projects_import_tracking.rb

```diff
@@ -0,0 +1,16 @@
+class ProjectsImportTracking < ActiveRecord::Migration[6.1]
+  def change
+    create_table :ac_hmis_projects_import_attempts do |t|
+      t.string :status, null: false, default: 'init'
+      t.string :etag, null: false, comment: 'fingerprint of the file'
+      t.text :key, null: false, comment: 'path in an s3 bucket to the file'
+      t.jsonb :result, null: false, default: {}
```

Comment:
This might is probably sufficient for storing some simple result counts, but we might want a place to log individual errors against the row that caused it.  What we do in the main importer:

drivers/hmis_csv_importer/app/models/hmis_csv_importer/importer/import_error.rb

Which has things like:
```
details: "PG::UniqueViolation: ERROR:  duplicate key value violates unique constraint "Assessment_pkey"
DETAIL:  Key (id)=(69) already exists."
source_type: "GrdaWarehouse::Hud::Assessment"
source_id: "8c8dec8fcaf2e9abeefb01686b92e3d3714eef3a9e5499d29a00b5b0ef0704c9"
``` 

Then we can surface that alongside the import log to help track down which items are causing the problem.

Reply:
I'll see if I can continue to use bulk import with validation and do that. Should I reuse that log table? I guess I might have to fall back to per-record A/R upserts.